### PR TITLE
Updated dependencies for dashboards

### PR DESCRIPTION
**Pull request by anmolsachan**
gluster-mixin-issue : #19 
Signed-off-by: Anmol Sachan <anmol13694@gmail.com>

### CODE REVIEW

**Comment by cloudbehl** [comment]
@anmolsachan Looks good to me, can you create a corresponding issue and tag.

**Comment by umangachapagain** [comment]
@anmolsachan @cloudbehl  This might crash the Makefile. I don't think it takes into account the `Vendor` thing that `jb` creates. Specifically for Dashboards. So might have to fix it.

**Comment by cloudbehl** [comment]
@umangachapagain Tested the PR Works fine it already takes vendor in account.

**Comment by cloudbehl** [comment]
@cloudbehl LGTM

@umangachapagain @shtripat can you review it once

File: README.md [hunk 1]
```diff
@@ -25,6 +28,11 @@ $ git clone https://github.com/gluster/gluster-mixins.git
 $ cd gluster-mixins
 ```
 
+**To get dependencies**
+
+`$ jb install`
```
[the remark this answers is not in the document]
Comment (anmolsachan) [reply README.md]: For this project we are not using the JB install. Just to init the dependencies for development. If required later we can add them if required.

[the remark this answers is not in the document]
Comment (shtripat) [reply README.md]: @umangachapagain do you ack this and can we approve this?

[the remark this answers is not in the document]
Comment (umangachapagain) [reply README.md]: In that case I assume that `vendor` folder wouldn't be checked in.

lgtm.

**Pull request merged**